cspell.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dictionaries": ["custom-words"],
+  "dictionaryDefinitions": [
+    {
+      "name": "custom-words",
+      "path": "./cspell-wordlist.txt",
+      "addWords": true
+    }
+  ],
+  "ignoreRegExpList": [
+    "/(```+)[\\s\\S]+?\\1/g",
+    "/`[a-zA-Z-_]+`/g"
```

Comment:
the first one here ignores multi-line code blocks, the second inline code blocks

Reply:
Oh JSON, why won't you let us add comments to things 🙄 

I don't think it's worth converting this to YAML or CJS, but do I wish they supported JSON5